### PR TITLE
[Merged by Bors] - chore: removed unused variables

### PR DESCRIPTION
**Pull request by adomani**
Part of/working towards #17715

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/IsAlgClosed/Classification.lean [hunk 1]
```diff
@@ -119,10 +119,9 @@ end Classification
 
 section Cardinal
 
-variable {R L K : Type u} [CommRing R]
+variable {R K : Type u} [CommRing R]
 variable [Field K] [Algebra R K] [IsAlgClosed K]
```
Comment (grunweg) [inline Mathlib/FieldTheory/IsAlgClosed/Classification.lean:123]: Combine this with the previous line? Optional.

File: Mathlib/RingTheory/AlgebraicIndependent.lean [hunk 2]
```diff
@@ -47,11 +47,10 @@ open scoped Classical
...
 
 variable {ι : Type*} {ι' : Type*} (R : Type*) {K : Type*}
-variable {A : Type*} {A' A'' : Type*} {V : Type u} {V' : Type*}
+variable {A : Type*} {A' : Type*}
 variable (x : ι → A)
-variable [CommRing R] [CommRing A] [CommRing A'] [CommRing A'']
-variable [Algebra R A] [Algebra R A'] [Algebra R A'']
-variable {a b : R}
+variable [CommRing R] [CommRing A] [CommRing A']
+variable [Algebra R A] [Algebra R A']
```
Comment (grunweg) [inline Mathlib/RingTheory/AlgebraicIndependent.lean:53]: Optional: combine these two lines?

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

**Review by Vierkantor — APPROVED** [review]
I'm happy with or without merging the lines.

bors d+

**Comment by adomani** [comment]
Thank you both!  I merged the lines and I will

Bors merge

now!

**Comment by Vierkantor** [comment]
I think

bors merge

is case sensitive, no?

**Comment by Vierkantor** [comment]
Sorry bors, you are not case sensitive after all! 👁️

**Comment by adomani** [comment]
I was precisely testing this: it seems that it is case-sensitive whether or not the label is added, but not whether the merging actually happens.

**Comment by Vierkantor** [comment]
Aha! Sorry for interfering in your test then.

**Comment by adomani** [comment]
Kim also had your [same reaction](https://github.com/leanprover-community/mathlib4/pull/12339#issuecomment-2425712946).  There, it was a "feature" of auto-complete on my phone, here it was a deliberate test!  :smile:

**Comment by adomani** [comment]
> Aha! Sorry for interfering in your test then.

No worries, the test had already taken place!

**Pull request closed without merging**